<launch>
    <node name="bodyvel_sensor" pkg="ssf_updates" type="bodyvel_sensor" clear_params="true" output="screen">
	    <remap from="ssf_core/hl_state_input" to="/auk/fcu/ekf_state_out" />
	    <remap from="ssf_core/correction" to="/auk/fcu/ekf_state_in" />

	    <remap from="ssf_core/bodyvel_measurement" to="/auk/camera/image_raw" />

	    <rosparam file="$(find ssf_updates)/bodyvel_sensor_fix.yaml"/>
	    <rosparam file="$(find iof)/IOFFixParams.yaml"/>
    </node>
</launch>
